src/osmf/js/osmf_flash.js

```diff
@@ -2,6 +2,11 @@
  * OSMF flash video plugin
  */
 
+require("../../../html5-common/js/utils/InitModules/InitOO.js");
+require("../../../html5-common/js/utils/InitModules/InitOOUnderscore.js");
+require("../../../html5-common/js/utils/InitModules/InitOOHazmat.js");
+require("../../../html5-common/js/utils/constants.js");
+require("../../../html5-common/js/utils/environment.js");
```

Comment:
Removed inclusion of enviroment.js.

uglify_fn encapsulated the code of osmf_flash.js inside another function due to which the HDSPlayer was not getting initialised. The issue appears as the external interface keeps on waiting for the javascript to get ready, as "isReady" function in osmf_flash.js is not being recognised.  As per the fix, binding "isReady" to the current object fixes the issue.